libs/test/SmartStoreTest/src/com/salesforce/androidsdk/store/SmartStoreTestCase.java

```diff
@@ -86,7 +93,72 @@ protected boolean hasTable(String tableName) {
 			safeClose(c);
 		}
 	}
-	
+
+	/**
+	 * Helper method to check columns of table
+	 * @param tableName
+	 * @param expectedColumnNames
+     */
+	protected void checkColumns(String tableName, List<String> expectedColumnNames) {
```

Comment:
A bunch of new helper methods:
- to check the columns in the database
- to check the database indexes
- to check the explain query plan of the last query run